packages/ckeditor5-list/tests/listediting.js

```diff
@@ -1646,7 +1646,15 @@ describe( 'ListEditing', () => {
 										'</ul>' +
 										'</li>' +
 										'</ul>' +
-										'<p>e</p>' +
```

Comment:
Why did you decide to modify this test?

Reply:
This test used nested tables so I didn't want to change the scope of this test. I just decided to update it to the changes that my PR introduces. Is it wrong?

Reply:
No, it's completely fine. I didn't understand the change at first, but it makes perfect sense.